media/CircleEditor/view-sidebar.js

```diff
@@ -1456,30 +1439,269 @@ sidebar.ArgumentView = class {
+                        }
+                        break;
+                    case 'int32':
+                        view.setInt32(0, parseInt(bufferArr[i]), true);
+                        for (let j = 0; j < 4; j++) {
+                            originalBuf[i * 4 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 'uint8':
+                        view.setUint8(0, parseUint(bufferArr[i]), true);
+                        originalBuf[i] = view.getUint8(0);
+                        break;
+                    case 'int64':
+                        view.setBigInt64(0, BigInt(parseInt(bufferArr[i])), true);
+                        for (let j = 0; j < 4; j++) {
+                            originalBuf[i * 4 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 'boolean':
+                        if (bufferArr[i] === 'false' || bufferArr[i] - 0 === 0) {
+                            originalBuf[i] = 0;
+                        } else {
+                            originalBuf[i] = 1;
+                        }
+                        break;
+                    case 'int16':
+                        view.setInt16(0, parseInt(bufferArr[i]), true);
+                        for (let j = 0; j < 2; j++) {
+                            originalBuf[i * 2 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 'int8':
+                        view.setInt8(0, parseInt(bufferArr[i]), true);
+                        originalBuf[i] = view.getUint8(0);
+                        break;
+                    case 'float64':
+                        view.setFloat64(0, parseFloat(bufferArr[i]), true);
+                        for (let j = 0; j < 8; j++) {
+                            originalBuf[i * 8 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 'uint64':
+                        view.setBigUint64(0, BigInt(parseInt(bufferArr[i])), true);
+                        for (let j = 0; j < 8; j++) {
+                            originalBuf[i * 8 + j] = view.getUint8(j);
+                        }
+                        break;
+                    case 'uint32':
+                        view.setUint32(0, parseInt(bufferArr[i]), true);
+                        for (let j = 0; j < 4; j++) {
+                            originalBuf[i * 4 + j] = view.getUint8(j);
+                        }
+                        break;
+                    default:  // TODO Enable other types
+                        break;
+                }
+            }
+        }
+
+        const newBuffer = [];
+
+        for (let i = 0; i < Object.keys(original).length; i++) {
+            newBuffer.push(original[i]);
+        }
+
+        this._editObject._arguments._initializer._data = newBuffer;
+    }
+
+    removeBracket(buffer) {
+        let str = buffer;
```

Comment:
I think it would be more simple..?

```suggestion
    removeBracket(str) {
```